### PR TITLE
Fix/update account creation page/flow

### PR DESCRIPTION
**Pull request by YouQam**
This PR closes this ticket [#284](https://github.com/mint-o-badges/badgr-ui/issues/284). It includes updating signup and verify-email-address pages

> ⚠️  I've created a new ticket for 'resend confirmation email' as it involves server implementation.

### CODE REVIEW

**Review by umut0 — COMMENTED** [review]
Do we need the "E-Mail erneut senden" Button?

The input field situation is a bit of a mess, need to fix that next.

Also fixed some typos.

**Comment by YouQam** [comment]
> Do we need the "E-Mail erneut senden" Button?
> 
> The input field situation is a bit of a mess, need to fix that next.
> 
> Also fixed some typos.

The `resend E-Mail` button was included in the design but I wasn't sure if they need it atm. Now it has a separate ticket.
Regarding the `input field` can you clarify what needs to be fixed?

**Comment by umut0** [comment]
> > Do we need the "E-Mail erneut senden" Button?
> > The input field situation is a bit of a mess, need to fix that next.
> > Also fixed some typos.
> 
> The `resend E-Mail` button was included in the design but I wasn't sure if they need it atm. Now it has a separate ticket. Regarding the `input field` can you clarify what needs to be fixed?

I am working on a new input field at the moment, so you dont need to worry about that. We will exchange all old implementations once its done.

File: src/app/signup/components/signup-success/signup-success.component.html [hunk 1]
```diff
@@ -1,16 +1,26 @@
...
-		<p class="u-text-body u-margin-top1x">Willkommen bei {{ service }}!</p>
-		<p class="u-text-body u-margin-top4x">
-			Wir haben Dir eine Verifizierungs-Email an <strong class="u-text-bold">{{ email }}</strong> geschickt.
-		</p>
-		<p class="u-text-body u-margin-top2x">
-			Klicke die URL, um den Registrierungsprozess abzuschließen. Wenn Du die Email nicht in wenigen Minuten
-			erhältst, schaue in Deinen Spam-Ordner. Wenn Du die Mail immer noch nicht findest, dann
-			<a class="u-text-link" [href]="helpEmailUrl">schreib uns</a>.
-		</p>
+<div class="oeb page-padding tw-overflow-hidden tw-relative md:tw-py-[6rem] tw-py-[2rem]">
```
[the remark this answers is not in the document]
Comment (YouQam) [reply src/app/signup/components/signup-success/signup-success.component.html:1]: I did test it with safari and it works fine. Should I do it anyway?

File: src/app/signup/components/signup/signup.component.html [hunk 2]
```diff
@@ -1,174 +1,222 @@
-<div class="l-containerxaxis l-containeryaxis l-flex l-flex-justifycenter">
-	<form-message></form-message>
+<div class="oeb page-padding tw-overflow-hidden tw-relative md:tw-py-[6rem] tw-py-[3rem]">
```
[the remark this answers is not in the document]
Comment (YouQam) [reply src/app/signup/components/signup/signup.component.html:1]: This as well.

File: src/app/signup/components/signup-success/signup-success.component.html (hunk 1 above)
[the remark this answers is not in the document]
Comment (umut0) [reply src/app/signup/components/signup-success/signup-success.component.html:1]: ok than just leave it like it is. I dont know why it caused problems on the other pages

[the remark this answers is not in the document]
Comment (YouQam) [reply src/app/signup/components/signup-success/signup-success.component.html]: I made the updates you suggested to avoid safari issues.

**Pull request merged**